### PR TITLE
Make accept-invite viewname configurable in django settings

### PR DESCRIPTION
**Pull request by BartoszCki**
This change allows to change the invitation link using Django settings

### CODE REVIEW

File: invitations/app_settings.py [hunk 1]
```diff
@@ -98,5 +98,12 @@ def ADMIN_CHANGE_FORM(self):
             "invitations.forms.InvitationAdminChangeForm"
         )
 
+    @property
+    def COMFIRMATION_VIEWNAME(self):
```
Comment (valberg) [inline invitations/app_settings.py:102]: Can we call it `CONFIRMATION_URL_NAME` since it's the name of the URL pattern we are referring to?

**Comment by valberg** [comment]
@BartoszCki Let us get this in! Can you deal with the conflicts and rename the setting as per my comment?

File: invitations/app_settings.py (hunk 1 above)
Comment (BartoszCki) [reply invitations/app_settings.py:102]: I've removed the source repository a long time ago and can't find a way to update this PR. I guess I'll have to create a new branch and new PR

Comment (valberg) [reply invitations/app_settings.py:102]: Roger that - maybe you can use the .patch as a starting point: https://github.com/jazzband/django-invitations/pull/108.patch

**Comment by BartoszCki** [comment]
Created new PR with the same changes
https://github.com/jazzband/django-invitations/pull/177

**Pull request closed without merging**